cmd/up/up.go

```diff
@@ -598,6 +598,9 @@ func (up *upContext) waitUntilExitOrInterruptOrApply(ctx context.Context) error
 
 func (up *upContext) applyToApps(ctx context.Context) chan error {
 	result := make(chan error, 1)
+	if up.Options.DockerDesktop {
```

Comment:
This is tricky. When running `okteto up` from the Docker Extension, maybe because they run with a max of goroutines, the cli process crashes if we start the `go tr.App.Watch` goroutines.
Since this isn't needed when running a compose file, I don't creating the watchers, but we should investigate what's the source issue